libs/index.md

```diff
@@ -66,7 +66,7 @@ Available query params:
 |![](/badge/chat/on%20gitter/cyan) | [/badge/chat/on%20gitter/cyan](/badge/chat/on%20gitter/cyan)
 |![](/badge/stars/★★★★☆/green) | [/badge/stars/★★★★☆](/badge/stars/★★★★☆/green)
 |![](/badge/become/a%20patron/F96854) | [/badge/become/a%20patron/F96854](/badge/become/a%20patron/F96854)
-|![](/badge/code%20style/standard/f2a) | [/badge/style/standard/f2a](/badge/style/standard/f2a)
+|![](/badge/code%20style/standard/f2a) | [/badge/style/standard%20style/f2a](/badge/style/standard%20style/f2a)
```

Comment:
I think this should be `/badge/code%20style/standard/f2a` to match the badge 😄

Reply:
Correct, will change it .